src/site/markdown/index.md

```diff
@@ -1,3 +1,70 @@
-# Documentation
+# WBEM Java Utility
```

Comment:
WBEM Java Client

Reply:
I'll check if other documents are impacted by the "WbeM" curse